lib/gui/app/components/wifi/react/keyboard/style.scss

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright 2018 resin.io
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+.on-screen-kb input {
+  font-family: Monospaced Number,Chinese Quote,-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,PingFang SC,Hiragino Sans GB,Microsoft YaHei,Helvetica Neue,Helvetica,Arial,sans-serif;
+  -webkit-box-sizing: border-box;
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  list-style: none;
+  position: relative;
+  display: inline-block;
+  padding: 4px 11px;
+  width: 100%;
+  height: 32px;
+  font-size: 14px;
+  line-height: 1.5;
+  color: rgba(0,0,0,.65);
+  background-color: #fff;
+  background-image: none;
+  border: 1px solid #d9d9d9;
+  border-radius: 4px;
+  -webkit-transition: all .3s;
+  transition: all .3s;
+}
+
+.ui-keyboard {
+  position: fixed;
+  z-index: 9999;
+  width: 100%;
+  left: 0;
+  bottom: 0;
+  text-align: center;
+  background-color: black;
+  padding: 0.5em;
+
+	.ui-keyboard-button {
+		background-color: #3e3e3e;
+		color: whitesmoke;
+    padding: 0.5em 1.5em;
+    margin: 0.2em;
+    border: 0;
+    border-radius: 0.1em;
+    font-size: 1.75em;
+    border-top: 1px solid #4e4e4e;
```

Comment:
Indentation needs fixing here

Reply:
This should be fixed, although didn't see any error/warning on my editor... They're all the same level. Gonna retry de-tab/re-tabbing them